bears/coffee_script/CoffeeLintBear.py

```diff
@@ -22,11 +23,280 @@ class CoffeeLintBear:
     CAN_DETECT = {'Syntax', 'Formatting', 'Smell'}
 
     severity_map = {'warn': RESULT_SEVERITY.NORMAL,
-                    'error': RESULT_SEVERITY.MAJOR}
+                    'error': RESULT_SEVERITY.MAJOR,
+                    'ignore': RESULT_SEVERITY.INFO}
 
     @staticmethod
     def create_arguments(filename, file, config_file):
-        return '--reporter=raw', filename
+        return '--reporter=raw', '-f', config_file, filename
+
+    @staticmethod
+    def generate_config(filename, file,
+                        max_line_length: int=79,
+                        max_line_length_affect_comments: bool=True,
+                        space_before_and_after_arrow: bool=True,
+                        check_braces_spacing: bool=False,
+                        braces_spacing_width: int=1,
+                        spacing_in_empty_braces: int=1,
+                        class_naming_camel_case: bool=True,
+                        spaces_before_colon: int=-1,
+                        spaces_after_colon: int=-1,
+                        enforce_newline_at_EOF: bool=True,
```

Comment:
so IMO we should take what users will take most likely. When in doubt just take their defaults because they probably know better. In this case I'm not sure but I'd go for True here, some editors even do it automatically for *every* file.